include/glow/Graph/Graph.h

```diff
@@ -863,37 +869,35 @@ class Function final : public Named {
   /// dimension of data indexed by the \p indices vector, and then accumulates
   /// them into len(\p lengths) entries: first Lengths[0] slices are aggregated
   /// to Result[0], next Lengths[1] slices are aggregated to Result[1], etc.
-  /// I.e. sum(Lengths) must be equal to len(Indices).
+  /// I.e. sum(Lengths) must be equal to len(Indices). \p precision represents
+  /// what precision to use for Scale, Offset, and Result.
   FusedRowwiseQuantizedSparseLengthsSumNode *
-  createFusedRowwiseQuantizedSparseLengthsSum(llvm::StringRef name,
-                                              Constant *data, NodeValue indices,
-                                              NodeValue lengths);
+  createFusedRowwiseQuantizedSparseLengthsSum(
+      llvm::StringRef name, Constant *data, NodeValue indices,
+      NodeValue lengths, ElemKind precision = ElemKind::FloatTy);
```

Comment:
nit: I mentally associate precision with the precision of the data which is uint8 in both cases. Maybe, scalePrecision for clarity.

Reply:
Hm, so precision here refers to precision for scale, offset, and result. Not sure what to name it accordingly except for something like `scaleOffsetResultPrecision` 🙃I can do that if you'd prefer, or if you have another suggestion.

Reply:
Let's leave it.